orchagent/fdborch.cpp

```diff
@@ -1147,7 +1161,7 @@ bool FdbOrch::addFdbEntry(const FdbEntry& entry, const string& port_name,
     }
 
     /* Retry until port is member of vlan*/
-    if (vlan.m_members.find(port_name) == vlan.m_members.end())
+    if (check_vlan_member && vlan.m_members.find(port_name) == vlan.m_members.end())
```

Comment:
I see that the EvpnRemoteVnip2mpOrch::addOperation already has a call to 
gPortsOrch->addVlanMember in which case I don't see why this check is required. 
if we call portsorch->isVlanMember.. It should work for both P2P and P2MP ?

Reply:
Actually I thought of a better logic to include checks for P2P and P2MP. In P2MP scenario, the vlan member differentiator is remote IP (since only one port exists based on SIP tunnel) and thus including that in check will  ensure it works in both scenarios.